### PR TITLE
feat(make): allows overwriting CONTAINER_ vars

### PR DESCRIPTION
**Pull request by bartoszmajsak**
With this change you can call `CONTAINER_BUILDER=docker make build` or rely on your `.env` file instead of editing `Makefile` by hand.

### CODE REVIEW

**Review by lucferbux — CHANGES_REQUESTED** [review]

File: Makefile
Comment (bartoszmajsak) [reply Makefile:13]: Thanks, not sure how I missed the `-e`  flag. TIL

https://media.giphy.com/media/w89ak63KNl0nJl80ig/giphy.gif

I will improve the docs as part of this PR.

Comment (lucferbux) [reply Makefile:13]: Perfect! Thank you very much!

Comment (lucferbux) [reply Makefile:13]: Hi @bartoszmajsak did you have the time to review this? If you want we can take a look at this, it would be nice to merge this as soon as possible cause it was a really nice improvement!
Thanks in advance!

Comment (bartoszmajsak) [reply Makefile:13]: @lucferbux I was away on vacation, I will finalize it this week.

Comment (lucferbux) [reply Makefile:13]: Great, thank you very much!

Comment (lucferbux) [reply Makefile:13]: @bartoszmajsak we are doing a big revamp of our docs right now, feel free to [take a look](https://github.com/opendatahub-io/odh-dashboard/pull/1851), rebase once it's merged and update accordingly, thanks a lot!

**Comment by andrewballantyne** [comment]
/hold
Not sure if this impact the stability of the release, but we just hit feature freeze.

We should look to get this in next sprint @lucferbux 

Also removing the wip label - I think that was just used like a hold.

**Comment by lucferbux** [comment]
> /hold Not sure if this impact the stability of the release, but we just hit feature freeze.
> 
> We should look to get this in next sprint @lucferbux
> 
> Also removing the wip label - I think that was just used like a hold.

@bartoszmajsak any update on this? If you don't feel confident enought to update the docs, we can get this merged and I can do a follow up updating the CONTRIBUTING guidelines.
Thanks in advance!

**Comment by bartoszmajsak** [comment]
I'm back from pto next week, will ping you @lucferbux

**Comment by lucferbux** [comment]
> I'm back from pto next week, will ping you @lucferbux

Hi @bartoszmajsak, happy new year!!

Could you rebase this and update the docs, once that's done we can merge this. Thanks in advance!

**Comment by bartoszmajsak** [comment]
@lucferbux ah this long forgotten little PR of mine. Thanks for keeping me pinged :)

Happy New Year 🎊 I'm back in action tomorrow so I'll put it on top of my queue as a warmup.

**Comment by lucferbux** [comment]
> @lucferbux ah this long forgotten little PR of mine. Thanks for keeping me pinged :)
> 
> Happy New Year 🎊 I'm back in action tomorrow so I'll put it on top of my queue as a warmup.

Great thanks a lot and enjoy your PTO!

**Comment by bartoszmajsak** [comment]
@lucferbux it's rebased and I added a note in the `CONTRIBUTING.md`. I hope now it's good enough.

**Comment by lucferbux** [comment]
> @lucferbux it's rebased and I added a note in the `CONTRIBUTING.md`. I hope now it's good enough.

Great, thanks a lot, this is just perfect!!

**Pull request merged**
